src/pages/Market/index.tsx

```diff
@@ -85,6 +85,7 @@ export const MarketPage = () => {
       isAuction: (timedAuction && fixedPrice) || (!timedAuction && !fixedPrice) ? undefined : timedAuction && !fixedPrice,
       ...prices,
       collectionId: collections,
+      numberOfAttributes: attributeCounts,
```

Comment:
Please don't forget to rename attributeCounts to numberOfAttributes on our side for consistency